docs/website/docs/general-usage/schema-contracts.md

```diff
@@ -151,19 +151,25 @@ Following tables are considered new:
 
 For example such table is considered new because column **number** is incomplete (define primary key and NOT null but no data type)
 ```yaml
-  blocks:
-    description: Ethereum blocks
-    write_disposition: append
-    columns:
-      number:
-        nullable: false
-        primary_key: true
-        name: number
+blocks:
+  description: Ethereum blocks
+  write_disposition: append
+  columns:
+    number:
+      nullable: false
+      primary_key: true
+      name: number
 ```
 
 What tables are not considered new:
 1. Those with columns defined by Pydantic modes
 
+### Working with datasets that have manually added tables and columns on the first load
+
+In some cases you might be working with datasets that have tables or columns created outside of dlt. If you are loading to a table not created by `dlt` for the first time, `dlt` will not know about this table while enforcing schema contracts. This means that if you do a load where the `tables` are set to `evolve`, all will work as planned. If you have `tables` set to `freeze`, dlt will raise an exception because it thinks you are creating a new table (which you are from dlts perspective). You can allow `evolve` for one load and then switch back to `freeze`. 
+
+The same thing will happen if `dlt` knows your table, but you have manually added a column to your destination and you have `columns` set to `freeze`.
```

Comment:
I had just one thought. We have `pipeline.sync_schema` method. What if instead of letting to evolve once we use it after adding a column? should work and will make stuff easier

Reply:
maybe that was misleading. this is an idea to check in the test you write. but if it looks confusing let's just close it